RedHat-UX/red-hat-design-system
# docs(skip link): add images to best practices

## What I did

Closes #1652 


## Testing Instructions

Check [`<rh-skip-links>` best practices](https://deploy-preview-1681--red-hat-design-system.netlify.app/elements/skip-link/guidelines/#best-practices) in DP for images

## Notes to Reviewers
